QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -16,13 +17,21 @@
 import ccre.ctrl.PIDControl;
 import ccre.ctrl.Ticker;
 import ccre.igneous.Igneous;
+import ccre.instinct.AutonomousModeOverException;
+import ccre.instinct.InstinctModule;
 import ccre.log.Logger;
 
 public class Clamp {
-    public static FloatStatus heightOrSpeed = new FloatStatus();
+    public static final boolean MODE_SPEED = true;
+    public static final boolean MODE_HEIGHT = false;
+    
+    public static FloatStatus height = new FloatStatus();
+    public static FloatStatus speed = new FloatStatus();
     public static BooleanStatus mode = new BooleanStatus(); // true = speed,
                                                             // false = height
```

Comment:
You can remove the comment now that you have fields for MODE_SPEED and MODE_HEIGHT.